src/coreclr/vm/ecalllist.h

```diff
@@ -280,6 +280,9 @@ FCFuncStart(gRuntimeAssemblyFuncs)
     FCFuncElement("GetToken", AssemblyHandle::GetToken)
 FCFuncEnd()
 
+//FCFuncStart(gAssemblyBuilderFuncs)
```

Comment:
This should be deleted instead of adding commented out dead code.

Reply:
Right, sorry was planning to delete but forgot 😛